Library/Homebrew/cmd/update-bash.sh

```diff
@@ -305,7 +323,7 @@ EOS
   wait
   trap - SIGINT
 
-  for DIR in "$HOMEBREW_REPOSITORY" "$HOMEBREW_LIBRARY"/Taps/*/*
+  for DIR in "${DIRS[@]}"
   do
     [[ -d "$DIR/.git" ]] || continue
     pull "$DIR"
```

Comment:
> cleaning up untracked files

I don't think `pull` will do this. Instead, it will stash then unstash untracked files. Therefore, I can only see it increase overhead.